Challenge-4/Bot_2/src/helper.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  CONFIGURATION_ABI,
+  CONFIGURATOR_PROXY,
+  CONFIGURATOR_PROXY_POLYGON,
+} from "./constants";
+import { AlertsResponse, ethers, getAlerts } from "forta-agent";
+import { AlertType } from "./types";
+import { LRUCache } from "lru-cache";
```

Comment:
This is unused, but the fact you imported it makes me thing you meant to implement some caching. Please do so, as I left in my comment.

However, the caveat is that I don't fully understand how the return values work, so if they are constantly changing, then caching probably wouldn't work as they would always be a different value from a block to the next. Though, to me, that sounds unlikely. 🤷